src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -2688,6 +2688,8 @@ private void HandleToggleLightAndLink()
 					LedLightStatusLabel.Image = Emulator.AsDriveLight().DriveLightOn
 						? _statusBarDiskLightOnImage
 						: _statusBarDiskLightOffImage;
+
+					LedLightStatusLabel.ToolTipText = Emulator.AsDriveLight().DriveLightIconDescription;

```

Comment:
Memoise: `var diskLEDCore = Emulator.AsDriveLight();` (why that wasn't done already I don't know)

Also there might be some way to avoid updating the label when it hasn't changed? `GetHashCode`? Or maybe WinForms does that internally already. I don't think any part of the app handles that at the moment though so don't worry about it.

Reply:
Like this? https://github.com/TASEmulators/BizHawk/pull/4022/commits/1857557e86fe731cd1ddf9bed96b17f5b2a5e8cc

Reply:
Fixed on `master` as 6d05b9874c3dc7d0946055d3181745294c447267 and rebased this onto that.